crates/aptos-dkg/src/pvss/chunky/public_parameters.rs

```diff
@@ -172,13 +149,18 @@ impl<E: Pairing> PublicParameters<E> {
             pp_elgamal: chunked_elgamal::PublicParameters::default(),
             pk_range_proof: dekart_univariate_v2::Proof::setup(
                 max_num_chunks_padded,
-                CHUNK_SIZE as usize,
+                ell as usize,
                 group_generators,
                 rng,
             )
             .0,
             G_2: hashing::unsafe_hash_to_affine(b"G_2", DST),
-            powers_of_radix: arkworks::powers_of_two(num_chunks_per_share),
+            ell,
+            table: dlog::table::build::<E::G1>(
+                chunked_elgamal::PublicParameters::<E>::default().G.into(),
```

Comment:
Can avoid repeating this call since you previously called `chunked_elgamal::PublicParameters::<E>::default()`